package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "scripts": {
     "figma": "node ./scripts/figma.js",
```

Comment:
Should we keep the old one yet?

Reply:
I thought this is using it in GitHub Actions, but I noticed the file was deleted recently. 
Since it seems unnecessary, I will remove it and add a section about Bun in the README.

Reply:
done with https://github.com/ubie-oss/ubie-icons/pull/36/commits/fc2b434059ca9c1a2b1fc342475bc6682cc2330f